com.gitfiend.GitFiend.yaml

```diff
@@ -0,0 +1,88 @@
+app-id: com.gitfiend.GitFiend
+runtime: org.freedesktop.Sdk
+runtime-version: '22.08'
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: gitfiend
+tags:
+  - proprietary
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=ssh-auth
+  - --filesystem=home
+  - --filesystem=xdg-run/gnupg:ro
+  - --share=network
+  - --device=dri
+modules:
+  - name: unappimage
+    buildsystem: simple
+    build-commands:
+      - make -C squashfs-tools -j ${FLATPAK_BUILDER_N_JOBS} install INSTALL_DIR=${FLATPAK_DEST}/bin
+    sources:
+      - type: git
+        url: https://github.com/refi64/unappimage
+        commit: d7f86f2a0d7ec3a69211125207d5f127386b849a
+
+  - name: gitfiend
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 gitfiend ${FLATPAK_DEST}/bin/gitfiend
+      - install -Dm755 apply_extra ${FLATPAK_DEST}/bin/apply_extra
+      - install -Dm644 ${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - install -Dm644 ${FLATPAK_ID}.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
+      - for size in 16 24 32 48 64 128 256 512;
+        do
+          rsvg-convert -w $size -h $size -f png -o $size.png ${FLATPAK_ID}.svg;
+          install -Dm644 $size.png /app/share/icons/hicolor/${size}x${size}/apps/${FLATPAK_ID}.png;
+        done
```

Comment:
```suggestion
```

You don't need to do that with a SVG.

Reply:
It wasn't clear to me reading the documentation, so I included it for good measure. Thanks for the tip!